src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,22 @@
+<Configuration name="ConfigTest" status="ERROR" monitorInterval="5">
+	<Appenders>
+		<Console name="LogToConsole" target="SYSTEM_OUT">
+			<PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+		</Console>
+		<File name="LogToFile" fileName="logs/app.log">
+			<PatternLayout>
+				<Pattern>%d %p %c:%L %m%n</Pattern>
+			</PatternLayout>
+		</File>
+	</Appenders>
+	<Loggers>
+		<Logger name="mate.academy" level="info" additivity="false">
+			<AppenderRef ref="LogToFile"/>
+			<AppenderRef ref="LogToConsole"/>
+		</Logger>
+		<Root level="error">
+			<AppenderRef ref="LogToFile"/>
+			<AppenderRef ref="LogToConsole"/>
+		</Root>
+	</Loggers>
+</Configuration>
```

Comment:
<img width="396" alt="image" src="https://user-images.githubusercontent.com/58032907/200837299-8ef928ae-6b57-4ce9-bb9a-27da31ff84ac.png">

Reply:
+Corrected in new commit.